host/4/republish.yml

```diff
@@ -62,6 +62,17 @@ steps:
     displayName: tag and push dotnet images
     continueOnError: false
 
+  - bash: |
+      set -e
+      docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8
+      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8 $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet8
+      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8 $TARGET_REGISTRY/base:$(TargetVersion)-dotnet8
```

Comment:
RM. No need to push -base image imo.  Its better to just push images to descriptive locations